packages/admin/test/e2e/cli.feature.ts

```diff
@@ -88,118 +95,98 @@ describe('CLI', () => {
 
     const sdkCliDiff = difference(allFunctions, allCommands);
     const uncoveredFunctions = [
-      // renamed to get-airnode-parameters (which calls this more generic version)
-      'get-airnode-parameters-and-block-number',
-      // renamed to get-endorsement-status
-      'requester-to-client-address-to-endorsement-status',
-      // covered by endorse-client and unendrose-client commands
-      'set-client-endorsement-status',
-      // explicitely not implemented (not useful as of now)
+      'airnode-to-xpub',
+      'sponsor-to-requester-to-sponsorship-status',
+      'set-sponsorship-status',
       'get-templates',
       'check-authorization-status',
       'check-authorization-statuses',
-      'default-authorizers',
       'fail',
       'fulfill',
       'fulfill-withdrawal',
       'make-full-request',
-      'make-request',
-      'owner',
-      'renounce-ownership',
+      'make-template-request',
       'request-with-id-has-failed',
-      'client-address-to-no-requests',
-      'requester-to-next-withdrawal-request-index',
-      'set-default-authorizers',
-      'transfer-ownership',
+      'requester-to-request-count-plus-one',
+      'sponsor-to-withdrawal-request-count',
+      'templates',
     ].sort();
     expect(sdkCliDiff).toEqual(uncoveredFunctions);
   });
 
-  it('derives the designated wallet', async () => {
-    const requester = alice.address;
-    const airnodeWallet = ethers.Wallet.createRandom().connect(provider);
-    const airnodeMnemonic = airnodeWallet.mnemonic.phrase;
-    const masterWallet = ethers.Wallet.fromMnemonic(airnodeMnemonic, 'm').connect(provider);
-    // Fund the master wallet - which will be used to set the airnode parameters
-    await deployer.sendTransaction({
-      to: masterWallet.address,
-      value: ethers.utils.parseEther('1'),
-    });
-    airnodeRrp = airnodeRrp.connect(airnodeWallet);
-    const airnodeId = await admin.setAirnodeParameters(airnodeRrp, []);
+  it('derives the sponsor wallet', async () => {
+    const sponsor = alice.address;
 
     // Derive the wallet using CLI and admin SDK
     const out = execCommand(
-      'derive-designated-wallet',
-      ['--providerUrl', PROVIDER_URL],
-      ['--airnodeRrp', airnodeRrp.address],
-      ['--airnodeId', airnodeId],
-      ['--requester', requester]
+      'derive-sponsor-wallet',
+      ['--mnemonic', airnodeWallet.mnemonic.phrase],
+      ['--sponsor', sponsor]
     );
 
     // Derive the wallet programatically
-    const designatedWallet = await deriveDesignatedWallet(airnodeWallet, requester);
+    const sponsorWallet = await deriveSponsorWallet(airnodeWallet, sponsor);
 
     // Check that they generate the same wallet address
-    expect(out).toBe(`Designated wallet address: ${designatedWallet.address}`);
+    expect(out).toBe(`Sponsor wallet address: ${sponsorWallet.address}`);
```

Comment:
While I was doing the initial refactor of this package I wasn't sure whether to have these ad-hoc messages prepended or not, so I kept them.

WDYT? Should we remove them and only output the raw data? (We could say what the command returns in the README and CLI help).

Reply:
I think the SDK might be used by another application but CLI seems to me that it's meant to be used by a user. Therefore I like the messages we are currently showing becuase they give more info on the value being returned. With that said if we ever support localization and multiple languages on the admin package then it will be more work.

Reply:
Ok.

In that case, write some message also in https://github.com/api3dao/airnode/pull/456/files#diff-27795c748e8300543cfa14348dc8f4789262138f8d88dcce430afbf765677a97R252

Reply:
Actually, now I remember there are other reasons to avoid that "additional ad-hoc info" https://github.com/api3dao/airnode/issues/275

But it's OK to leave this as is for now.